views/login.handlebars

```diff
@@ -56,8 +56,8 @@
                         <label for="password" class="cols-sm-2 control-label">Password</label>
                         <div class="cols-sm-10">
                             <div class="input-group">
-                                <span class="input-group-addon"><i class="fa fa-lock fa-lg" aria-hidden="true"></i></span>
-                                <input type="password" class="form-control" name="password" id="password" placeholder="Enter your password" />
+                                <span class="input-group-addon" data-toggle="tooltip" title="A strong password contains at least 8 characters, 1 number, 1 capital letter and 1 symbol"><i class="fa fa-lock fa-lg" aria-hidden="true"></i></span>
```

Comment:
I think this tooltip should be moved to the bottom. So have like: Strength<sup>?</sup> and when you hover over the question mark, it shows the message. 

Also I suggest the following modification:
Very Weak: length < 8
Weak: >8 && lowercase only
Fair: >8 && lowercase && (upper || symbom || num)
Strong: >8 && lowercase && upper && (symbol || num)
Very Strong: >8 lowercase && upper && symbol && num